package.json

```diff
@@ -144,5 +144,9 @@
     "storybook-build": "build-storybook -c .storybook -o .storybook-out && sed -i -e 's/\\/sb_dll/sb_dll/g' .storybook-out/index.html",
     "snapshot": "build-storybook && PERCY_TOKEN=$STORYBOOK_PERCY_TOKEN PERCY_PROJECT=$STORYBOOK_PERCY_PROJECT percy-storybook --widths=1280",
     "webpack-profile": "yarn -s webpack --profile --json > stats.json"
+  },
+  "toolchain": {
+    "node": "8.10.0",
```

Comment:
Is this now also duplicating .nvmrc?

Reply:
Yes. Notion only reads `package.json`. But nvm drives me absolutely crazy which it's slowness but I'm not sure how many others are frustrated enough by nvm to make the switch.

Reply:
I'm just going to assume we're not talking about Notion.so here. I have no idea.

Reply:
Nope, it's this nvm replacement: https://www.notionjs.com/